manifest.json

```diff
@@ -1370,6 +1370,14 @@
       "path": "plugins/wordcount.lua",
       "id": "wordcount",
       "mod_version": "3"
+    },
+    {
+      "id": "www",
+      "description": "A simple library that provides a web client to fetch, and submit data via HTTP requests.",
+      "version": "0.1",
+      "type": "library",
+      "remote": "https://github.com/adamharrison/lite-xl-www.git:aecc866c33b0a954fbd41a5de2842bb97f4ac45e",
```

Comment:
Could the `.git` part of the URL cause issues?

Reply:
Don't see how; that is part of the official URL as provided by github.

Reply:
It's just that some plugins are using `.git`, others aren't. I guess it's fine.

Reply:
Odd. Github probably handles it the same either way, if it starts getting requests from a git client. But technically, the `.git` is official.